controllers/add_hserver.go

```diff
@@ -129,6 +130,16 @@ func (a addHServer) getContainer(hdb *hapi.HStreamDB) []corev1.Container {
 	container := corev1.Container{
 		Image:           hdb.Spec.HServer.Image,
 		ImagePullPolicy: hdb.Spec.HServer.ImagePullPolicy,
+		StartupProbe: &corev1.Probe{
+			ProbeHandler: corev1.ProbeHandler{
+				TCPSocket: &corev1.TCPSocketAction{
+					Port: intstr.FromString("port"),
+				},
+			},
+			InitialDelaySeconds: 5,
+			TimeoutSeconds:      2,
+			FailureThreshold:    3,
```

Comment:
Maybe we can set it in default web hook, and allow user modify it, because sometimes the application is not ready soon

Reply:
We have no need to integrate webhook just for setting default value to add complexity. By the way, user also can define this value by cr

Reply:
> user also can define this value by cr

How do it ? this looks hard code ?

Reply:
`	container := corev1.Container{
		Image:           hdb.Spec.HServer.Image,
		ImagePullPolicy: hdb.Spec.HServer.ImagePullPolicy,
		StartupProbe: &corev1.Probe{
			ProbeHandler: corev1.ProbeHandler{
				TCPSocket: &corev1.TCPSocketAction{
					Port: intstr.FromString("port"),
				},
			},
			InitialDelaySeconds: 5,
			TimeoutSeconds:      2,
			FailureThreshold:    3,
		},
	}

	structAssign(&container, &hServer.Container)`

the function `structAssign` will cover the default value if user defined

Reply:
Great !